doc/manuals/admin/management_api.md

```diff
@@ -31,7 +31,6 @@ A sample <i>statistics</i> response:
 
 ```
  <orion>
-   <xmlRequests>16</xmlRequests>
```

Comment:
Actually, the whole fragment should be changed to JSON, but not as part of this PR.

NTC